includes/definitions/packetshaper.yaml

```diff
@@ -1,6 +1,12 @@
 os: packetshaper
-text: 'Blue Coat Packetshaper'
+text: 'Blue Coat PacketShaper'
 type: network
+ifname: 1
+icon: bluecoat
+mib_dir:
+    - bluecoat
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
 discovery:
-    - sysDescr:
-        - PacketShaper
+    - sysObjectID:
+        - .1.3.6.1.4.1.14501.11.1.1
```

Comment:
According to our test data this will also need:

`.1.3.6.1.4.1.2334.1.1.`

You probably should also change `.1.3.6.1.4.1.14501.11.1.1` to `.1.3.6.1.4.1.14501.11.1.`

Reply:
Hi laf, .1.3.6.1.4.1.2334.1.1 old Packetshaper (EOL models). The newer S-Series appliances don't support this OID.

Reply:
Doesn't matter if it's EOL. We have people who use them still as they show up in our stats so they need to be supported.

Reply:
@laf Thanks for the comment. Would welcome your suggestions on fixing the fact that the S-Series Packetshaper are detected as Generic. 

![ps](https://user-images.githubusercontent.com/25568510/39783741-ef20b1bc-530d-11e8-9711-5b846ba09f4f.PNG). 

If the YAML file is changed, it's correctly detected.

Reply:
Changed to what though? Show what it is you've tried.

Reply:
On an amended LibreNMS install (live), it displays as - 

![screen shot 2018-05-08 at 22 49 05](https://user-images.githubusercontent.com/25568510/39784991-602b03ae-5312-11e8-8389-85747bc9aa9f.png)

The YAML file is - 

os: packetshaper
text: 'Blue Coat PacketShaper'
type: network
ifname: 1
icon: bluecoat
mib_dir:
    - bluecoat
over:
    - { graph: device_bits, text: 'Device Traffic' }
discovery:
    - sysObjectID:
        - .1.3.6.1.4.1.14501.11.1.1

Reply:
Those don't look like they are being detected as generic, just the logo didn't show in the first one.

Reply:
Hi @laf thats the point. With the S-Series appliances, the original packetshaper.yaml file doesn't match the appliance and thus change the logo (it shows as generic), but it is a Packetshaper. We have an older PS1400 in our lab, I'll test the file against this also, as it would cover both old and new.